crosswalks/Composer.csv

```diff
@@ -0,0 +1,68 @@
+Property,Packagist
+codeRepository,support.source
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,type
+applicationSubCategory,
+downloadUrl,
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,authors
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,
+datePublished,
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,keywords
+license,license
+producer,
+provider,
+publisher,
+sponsor,
+version,version
+isAccessibleForFree,
+isPartOf,
+hasPart,
+position,
+description,description
+identifier,name
+name,name
+sameAs,
+url,homepage
+relatedLink,
+givenName,
+familyName,
+email,author.email
+affiliation,
+identifier,
+name,author.name

```

Comment:
I don't see an `author` field in https://getcomposer.org/doc/04-schema.md ; I think these should just be:

```

email,email
affiliation,
identifier,
name,name
```

as `email` in an item of Composer's `authors` maps to `email` in an item of CodeMeta's `author`.

Also, huh, I just realized `name` is duplicated. (the one at like 46 for SoftwareApplication name, and this one for Person/Organization name.) Other crosswalks have the same issue, should we do something about them?

(and Composer has a specific field `support.email` for the SoftwareApplication's email, which we can't really express here)

Reply:
There is no `email` field in the [schema](https://getcomposer.org/doc/04-schema.md). The only options I can see are those you have mentioned (`authors.email` or `support.email`).
I think `support.email` is apt to the codemeta `email` field. However the description in [crosswalk.csv](https://github.com/codemeta/codemeta/blob/master/crosswalk.csv) doesn't make this so clear.

Reply:
@cboettig & @mbjones: we need your expertise also here